drivers/misc/nordic_vpr_launcher/nordic_vpr_launcher.c

```diff
@@ -38,6 +41,9 @@ static int nordic_vpr_launcher_init(const struct device *dev)
 	}
 #endif
 
+#if CONFIG_SOC_NRF54L15_ENGA_CPUAPP
```

Comment:
nit: configuring SPU should not take place if building for NSPE:
```suggestion
#if defined(CONFIG_SOC_NRF54L15_ENGA_CPUAPP) && !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
```